lib/extensions.js

```diff
@@ -24,7 +24,7 @@ module.exports = function(realmConstructor) {
 
     // Add sync methods
     if (realmConstructor.Sync) {
-        realmConstructor.Sync.User = require('./sync').User;
```

Comment:
Please remove this change as this conflicts with coming changes in the `al-sync-user` branch.